apps/system/js/app_usage_metrics.js

```diff
@@ -869,6 +941,11 @@
       var usage = new UsageData();
       if (data) {
         usage.data = data;
+        //Handle a scenario with old app data that does not have searches
+        if (typeof usage.data.searches === undefined) {
```

Comment:
This condition should never return true, I think you want either:
`if (typeof usage.data.searches === 'undefined')`, or..
`if (usage.data.searches === undefined)`, or even briefer.. :)
`if (!usage.data.searches)`